AdyenCard/Components/Card/CardComponent.swift

```diff
@@ -49,6 +49,9 @@ public class CardComponent: PublicKeyConsumer,
     /// Card component configuration.
     public let configuration: Configuration
     
+    /// Card component address builder.
+    public let addressViewModelBuilder: AddressViewModelBuilder
```

Comment:
maybe this should be `internal` or `private`

Reply:
yes, you are right I missed this. I have updated the code. Thanks for pointing it out.